# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/dex/hotfix-instal-t.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- dex-0.10.1/Makefile.vanilla	2026-07-30 14:11:22.332478047 +0300
+++ dex-0.10.1/Makefile	2026-07-30 14:12:26.587988713 +0300
@@ -13,13 +13,12 @@
 	@sphinx-build -b man -D version=$(TAG) -E man . $+
 
 .PHONY: install
-install: dex dex.1 README.rst LICENSE
+install: dex dex.1
 	@echo installing executable file to $(DESTDIR)$(PREFIX)/bin
 	@mkdir -p $(DESTDIR)$(PREFIX)/bin
 	@install -m 0755 $< $(DESTDIR)$(PREFIX)/bin/$(NAME)
 	@echo installing docs to $(DESTDIR)$(DOCPREFIX)
 	@mkdir -p $(DESTDIR)$(DOCPREFIX)
-	@install -m 0644 -t $(DESTDIR)$(DOCPREFIX)/ README.rst LICENSE
 	@echo installing manual page to $(DESTDIR)$(MANPREFIX)/man1
 	@mkdir -p $(DESTDIR)$(MANPREFIX)/man1
 	@install -m 0644 dex.1 $(DESTDIR)$(MANPREFIX)/man1/$(NAME).1

